src/main/java/at/ac/tuwien/damap/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -503,155 +530,120 @@ public void repoinfoAndToolsInformation() {
-
-        //Section 4c: ethical issues
-
-        log.debug("ethical part");
-
-        String ethicalIssues = "";
-        String ethicalStatement = "";
-        String otherEthicalIssues = "";
-        String committeeReviewed = "";
-
-        if (dmp.getHumanParticipants() != null) {
-            if (dmp.getHumanParticipants()) {
-                ethicalStatement = " " + loadResourceService.loadVariableFromResource(prop,"ethical.avail");
+            if (!datasetList.isEmpty()) {
+                legalRestrictionText += loadResourceService.loadVariableFromResource(prop, "legalDataset") + " ";
+                legalRestrictionText += multipleVariableAnd(datasetList) +". ";
             }
-        }
 
-        if (dmp.getEthicalIssuesExist() != null) {
-            if (dmp.getEthicalIssuesExist()) {
-                otherEthicalIssues = " " + loadResourceService.loadVariableFromResource(prop,"ethicalOther");
+            //add legal restrictions comment if available
+            if (dmp.getLegalRestrictionsComment() != null && !dmp.getLegalRestrictionsComment().isEmpty()) {
+                legalRestrictionText += loadResourceService.loadVariableFromResource(prop, "legalComment") +
+                                        " " + dmp.getLegalRestrictionsComment();
             }
-        }
 
-        if (dmp.getCommitteeReviewed() != null) {
-            if (dmp.getCommitteeReviewed()) {
-                committeeReviewed = " " + loadResourceService.loadVariableFromResource(prop,"ethicalReviewed.avail");
-            }
-            else {
-                committeeReviewed = " " + loadResourceService.loadVariableFromResource(prop,"ethicalReviewed.no");
+            if (dmp.getDataRightsAndAccessControl() != null && !dmp.getDataRightsAndAccessControl().isEmpty()) {
+                legalRestrictionText += ";" + dmp.getDataRightsAndAccessControl() + " " +
+                                            loadResourceService.loadVariableFromResource(prop, "legalRights.contact");
+            } else {
+                legalRestrictionText += ";" + loadResourceService.loadVariableFromResource(prop, "legalRights.contact.default");
             }
+        } else {
+            legalRestrictionText = loadResourceService.loadVariableFromResource(prop, "legal.no");
         }
+        addReplacement(replacements, "[legalrestriction]", legalRestrictionText);
+    }
 
-        String ethicalSentence = loadResourceService.loadVariableFromResource(prop,"ethicalStatement");
-
-        ethicalIssues = ethicalStatement + otherEthicalIssues + committeeReviewed;
-
-        if (!ethicalIssues.equals("")) {
-
-            ethicalIssues = ethicalSentence + ethicalIssues;
-
-            if (ethicalIssues.charAt(ethicalIssues.length()-1) == ' ')
-                ethicalIssues = ethicalIssues.substring(0,ethicalIssues.length()-1);
+    //Section 4c: ethical issues
+    public void ethicalIssuesText() {
+        log.debug("ethical part");
```

Comment:
remove log

Reply:
i think it is fine, we have several and it is only for debug build